model.h

```diff
@@ -1,35 +1,39 @@
-#ifndef MODEL_H
-
-#define MODEL_H
-
-// class for dealing with program logic
-#include "atomtree.h"
-#include "space.h"
-#include <iostream>
-#include <vector>
-#include <unordered_map>
-
-class AtomTree;
-struct Atom;
-class Space;
-class Model{
-  public:
-    void readRadiiAndAtomNumFromFile(std::string&);
-    inline double findRadiusOfAtom(const std::string&);
-    inline double findRadiusOfAtom(const Atom&); //TODO has not been tested
-    void readAtomsFromFile(std::string&);
-    // calls the Space constructor and creates a cell containing all atoms. Cell size is defined by atom positions
-    void defineCell(const double&, const int&);
-    void storeAtomsInTree();
-    void findCloseAtoms(const double&); //TODO
-    void calcVolume();
-    void debug();
-  private:
-    std::vector<Atom> atoms;
-    AtomTree atomtree;
-    Space cell;
-    std::unordered_map<std::string, double> radii;
-    std::unordered_map<std::string, int> elem_Z; //I wanted to use unsigned int for atomic number but stoui function doesn't exist, so it is simpler to use int
-};
-
-#endif
+#ifndef MODEL_H
+
+#define MODEL_H
+
+// class for dealing with program logic
+#include "atomtree.h"
+#include "space.h"
+#include <iostream>
+#include <vector>
+#include <map>
+#include <unordered_map>
+
+class AtomTree;
+struct Atom;
+class Space;
+class Model{
+  public:
+    void readRadiiAndAtomNumFromFile(std::string&);
+    inline double findRadiusOfAtom(const std::string&);
+    inline double findRadiusOfAtom(const Atom&); //TODO has not been tested
+    void listAtomTypesFromFile(std::string&);
+    void readAtomsFromFile(std::string&);
+    // calls the Space constructor and creates a cell containing all atoms. Cell size is defined by atom positions
+    void defineCell(const double&, const int&);
+    void storeAtomsInTree();
+    void findCloseAtoms(const double&); //TODO
+    void calcVolume();
+    void debug();
+    // following maps were moved to public for convenience but we can find alternatives if they need to be private

```

Comment:
The typical "alternative" to access a private member is called a "getter function". Generally, it's best to keep access to members exclusive to the class. For instance, the controller doesn't really need access to `number_of_atoms`, only to the `atoms_for_list` vector of tuples. Therefore, it makes sense to write a function in the Model class that returns `atoms_for_list` to the controller.